docs/getting-started/global-filter.md

```diff
@@ -19,6 +19,14 @@ Example with global filter `#task`:
 - [ ] #task take out the trash
 ```
 
+If your global filter is a tag, it supports subtags.
+
+Example with global filter `#task`:
+
+```markdown
+- [ ] #task/chores take out the trash
+```
+
```

Comment:
I don't understand the intention of this.

I think it is because the word `supports` isn't specific. Supports for what? In what way?

There is a markdown example, but it does not explain the significance of the task. At a guess, maybe it is saying that as well as tasks with `#task` matching a global filter `#task`, tasks such as `#task/chores` would also be treated by Tasks as having the global filter, and so would be recognised by the Tasks plugin.

Reply:
I think you are right - this addition to the doc is probably not needed / should be moved and it's confusing when added here. I'll remove it. 😄 Maybe I'll just add another example just above with a subtag.

---

What I was trying to get at is that the previously [this issue](https://github.com/obsidian-tasks-group/obsidian-tasks/discussions/365) prevented folks from being able to use subtags on their global filter tag.

Additionally (when I originally implemented this PR) Queries had no support for tags, so you could not navitely (for example) sort tasks by the subtags on your global filter. Now, of course, the tag query/sort/filter/group work great for this purpose.